components/scream/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -407,6 +410,15 @@ void linear_interp(SHOCLinearintData& d)
   d.transpose<ekat::util::TransposeDirection::f2c>();
 }
 
+void shoc_diag_second_moments_lbycond(SHOCSecondMomLbycondData& d)
+{
+  shoc_init(d.nlev, true);
+  d.transpose<ekat::util::TransposeDirection::c2f>();
```

Comment:
Since the data is 1-dimensional, we don't need to transpose anything, although it's harmless to make the call.